frontend/src/app/page.tsx

```diff
@@ -25,13 +25,17 @@ const navigation = [
   // { name: 'Reports', href: '#', icon: ChartPieIcon, current: false },
 ]
 const teams = [
-  { id: 1, name: 'CS 4485', href: '#', initial: 'C', current: false },
-  { id: 2, name: 'CS 4365', href: '#', initial: 'C', current: false },
-  { id: 3, name: 'CS 4314', href: '#', initial: 'C', current: false },
-]
+  { id: 1, name: 'CS 4485', initial: 'C', current: false },
+  { id: 2, name: 'CS 4365', initial: 'C', current: false },
+  { id: 3, name: 'CS 4314', initial: 'C', current: false },
+].map(team => ({
+  ...team,
+  href: `/class/${team.name.split(' ')[1]}`, // because the class page uses the class id number
```

Comment:
I think here you want to keep the letters and the numbers so the route should be /class/cs-4485 or something